TESTS/mbed_hal/qspi/flash_configs/N25Q128A_config.h

```diff
@@ -0,0 +1,226 @@
+    memset(reg_data, 0, QSPI_CONFIG_REG_2_SIZE);                                \
+    if (read_register(QSPI_CMD_RDCR2, reg_data,                                 \
+            QSPI_CONFIG_REG_2_SIZE, qspi) != QSPI_STATUS_OK) {                  \
+        return QSPI_STATUS_ERROR;                                               \
+    }                                                                           \
+                                                                                \
+    return ((reg_data[0] & CONFIG2_BIT_DE) == 0 ?                               \
+                QSPI_STATUS_OK : QSPI_STATUS_ERROR)
+
+
+#define DUAL_DISABLE_IMPLEMENTATION()                                           \
+                                                                                \
+    uint8_t reg_data[QSPI_CONFIG_REG_2_SIZE];                                   \
+                                                                                \
+    memset(reg_data, 0, QSPI_CONFIG_REG_2_SIZE);                                \
+    if (read_register(QSPI_CMD_RDCR2, reg_data,                                 \
+            QSPI_CONFIG_REG_2_SIZE, qspi) != QSPI_STATUS_OK) {                  \
+        return QSPI_STATUS_ERROR;                                               \
+    }                                                                           \
+    if (write_enable(qspi) != QSPI_STATUS_OK) {                                 \
+        return QSPI_STATUS_ERROR;                                               \
+    }                                                                           \
+    WAIT_FOR(WRSR_MAX_TIME, qspi);                                              \
+                                                                                \
+    reg_data[0] = reg_data[0] | (CONFIG2_BIT_DE);                               \
+    if (write_register(QSPI_CMD_WRCR2, reg_data,                                \
+            QSPI_CONFIG_REG_2_SIZE, qspi) != QSPI_STATUS_OK) {                  \
+        return QSPI_STATUS_ERROR;                                               \
+    }                                                                           \
+    WAIT_FOR(WRSR_MAX_TIME, qspi);                                              \
+    memset(reg_data, 0, QSPI_CONFIG_REG_2_SIZE);                                \
+    if (read_register(QSPI_CMD_RDCR2, reg_data,                                 \
+            QSPI_CONFIG_REG_2_SIZE, qspi) != QSPI_STATUS_OK) {                  \
+        return QSPI_STATUS_ERROR;                                               \
+    }                                                                           \
+                                                                                \
+    return ((reg_data[0] & CONFIG2_BIT_DE) == 1 ?                               \
+                QSPI_STATUS_OK : QSPI_STATUS_ERROR)
+
+
+#define QUAD_ENABLE_IMPLEMENTATION()    return QSPI_STATUS_OK
```

Comment:
What do these 3 macros do ? enable implementation = return STATUS OK?

Reply:
As I wrote this PR is IN DEVELOPMENT, implementation will be added soon

Reply:
I missed that line, would wait with review :)